galasa-managers-parent/galasa-managers-logging-parent/dev.galasa.elasticlog.manager/src/main/java/dev/galasa/elasticlog/internal/package-info.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * ElasticLog Manager
+ * 
+ * @galasa.manager ElasticLog
+ * 
+ * @galasa.release.state ALPHA - This Manager has been written to provide a way of exporting test data to ElasticSearch.  It has the bare 
+ * minimum code necessary to do that.  It has not been extensively tested.  The TPI of the LoggingManager interface is subject to change. 
+ * It can be used within tests.
+ * 
+ * @galasa.description
+ * 
+ * This Manager provides an interface to allow test data to be exported to an ElasticSearch endpoint. This allows for test data to be 
```

Comment:
This manager is used to export test results to an elastic search endpoint, where the data can be visualised on a Kabana dashboard.   Other managers can contribute to the information being exported to Elastic.